package.json

```diff
@@ -14,6 +14,7 @@
   "license": "Apache-2.0",
   "repository": "http://github.com/kruppel/check-engines",
   "dependencies": {
+    "cross-spawn": "^5.0.1",
```

Comment:
In addition to the shell support on node < v6 issue you mentioned, I found that we would need to detect the platform and conditionally run "npm" or "npm.cmd". cross-spawn does even better by supporting `PATHEXT`. All in all it felt like I would be re-engineering a solution that was already solved by the dependency, so I feel that it's worthwhile to pull it in given that it requires so little code change.

Regarding the version range - the carat is generally safer than `>=` because it won't pull in a version of cross-spawn that has breaking changes.